src/utilities/time.go.tmpl

```diff
@@ -31,6 +31,17 @@
 
 {{ $marker := "night" }}
 {{ $output := exec "weather" $location }}
+
+{{ if reFind `(?i)sorry, we are running out of queries to the weather service at the moment` $output }}
```

Comment:
`inFold`?

Reply:
Oh, duh -- had reFind during quick tests and not removed it :^)